Library/Homebrew/exceptions.rb

```diff
@@ -235,8 +235,10 @@ def dump
           puts "#{formula} was moved to homebrew-boneyard because it has unfixable issues."
           puts "Please do not file any issues about this. Sorry!"
         else
-          puts "If reporting this issue please do so at (not Homebrew/homebrew):"
-          puts "  https://github.com/#{formula.tap}/issues"
+          if issues_url = formula.tap.issues_url
+            puts "If reporting this issue please do so at (not Homebrew/homebrew):"
+            puts "  #{issues_url}"
```

Comment:
Yes it’ll be blank, so users will report issues on our main repo. I think it’s fine as a default; we might want to add special cases for known remote URLs (e.g. GitHub/Bitbucket/etc) in the future if we get too many unrelated issues.